language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -329,6 +331,75 @@ public static TypeSymbol getRawType(TypeSymbol typeDescriptor) {
         }
         return typeDescriptor;
     }
+    
+    public static boolean isErrorOrUnionOfErrors(TypeSymbol type) {
```

Comment:
Can we document this function since it's a public method in CommonUtil?

Reply:
Added